### ISSUE TITLE
option to set no border for floating clients

### ISSUE THREAD

**Issue by primalmotion** [vaxerski/Hyprland#222]
I would like to be able to tell that I don't want any border for floating containers.

**Comment by vaxerski** [comment]
added in 0f1ad16aecfdf89dd5d83d38318de8518157e6ee

**Comment by primalmotion** [comment]
thank you! It works. However there's still the blur being visible, which makes it look weird
![screenshot-2022-06-21-15:17:57](https://user-images.githubusercontent.com/208711/174906238-ed4a02e1-eac4-467d-b175-ba23deb38ca2.png)

**Comment by vaxerski** [comment]
![image](https://user-images.githubusercontent.com/43317083/174907505-8dea90f0-3889-4132-ae28-0bda6c0d347f.png)

cant reproduce.

Is that a GTK window? if so, that's the shadow. It's a bug, indeed, see the [FAQ entry](https://github.com/vaxerski/Hyprland/wiki/FAQ#gtk-apps-borders-are-weird-and-mouse-is-off)

**Comment by primalmotion** [comment]
Yeah it's a GTK 4 window (Shortwave) I already have the css trick but that does not seem to work for this one.

**Comment by vaxerski** [comment]
some apps force their own themes. You just gotta live with this until I fix that.

**Comment by primalmotion** [comment]
no problem :) thanks!

**Comment by vaxerski** [comment]
fixed in 770bada5d541e737ae17744f70eb019494471a29

**Comment by primalmotion** [comment]
that was quick! Thanks
